docs(cli): deprecate Kilo Console (#12701)
Kilo-Org/kilocode: packages/kilo-docs/pages/code-with-ai/platforms/cli.md

For detailed help on every command and subcommand, see the [CLI Command Reference](/docs/code-with-ai/platforms/cli-reference).

{% callout type="warning" title="Kilo Console is deprecated" %}

> The `kilo console` command and its browser interface are deprecated and may be removed in a future release. Use the CLI TUI, VS Code extension, or JetBrains plugin to run sessions. Edit Kilo configuration files directly to manage settings.

Author: ```suggestion
The `kilo console` command and its browser interface are deprecated and will be removed in a future release. Use the CLI TUI, VS Code extension, or JetBrains plugin to run sessions. Use the CLI slash commands, other extensions, or edit Kilo configuration files directly to manage settings.
```

> CLI attention alerts are disabled by default. Enable and configure them in either of these ways:

Author: ```suggestion
Enable and configure them by editing the TUI configuration:
```
